wandb/sdk/internal/internal_api.py

```diff
@@ -1729,8 +1746,12 @@ def create_artifact(
             # For backwards compatibility with older backends that don't support
             # distributed writers.
             (
-                "$distributedID: String" if distributed_id else "",
-                "distributedID: $distributedID" if distributed_id else "",
+                "$distributedID: String," if distributed_id else "",
+                "$clientID: ID!," if can_handle_client_id else "",
+                "$sequenceClientID: ID!," if can_handle_client_id else "",
```

Comment:
Client id is required here, but is nullable on line 1672. This broke the callsite [here](https://github.com/wandb/wandb/blob/1208a553b4aaa75926e379168138a411d578121e/wandb/apis/public.py#L2365). Repro:
```
run = wandb.Api().run(run_name)
artifact = wandb.Api().artifact(artifact_name)
run.log_artifact(artifact)
```